src/hooks/VoiceRecorder/index.tsx

```diff
@@ -34,11 +33,25 @@ const VoiceRecorderContext = createContext<VoiceRecorderContext>({
 export const VoiceRecorderProvider = (props: VoiceRecorderProps): React.ReactElement => {
   const { children } = props;
   const { config } = useSendbirdStateContext();
-  const { logger } = config;
+  const { logger, isVoiceMessageEnabled } = config;
   const [mediaRecorder, setMediaRecorder] = useState<MediaRecorder>(null);
   const [isRecordable, setIsRecordable] = useState<boolean>(false);
 
+  const [webAudioUtils, setWebAudioUtils] = useState(null);
```

Comment:
Hmm, here, maybe it was better to have a simple audioUtil, that doesn't convert file to mp3

Reply:
cc @HoonBaek